app/src/main/java/com/example/cartoon/Notification.kt

```diff
@@ -55,4 +65,9 @@ class Notification(private val context: Context) {
             manager.createNotificationChannel(channel)
         }
     }
+
+    companion object {
+        private const val PERMISSION_REQUEST_CODE = 1
```

Comment:
we can have a better request code. ex: 171023. Just in case.

Reply:
fine i wil do this